eclipsePlugin/build.gradle

```diff
@@ -18,28 +18,36 @@ sourceSets {
 
 configurations {
   embeddedLibs
+  includedLibs
   compile.extendsFrom embeddedLibs
+  compile.extendsFrom includedLibs
 }
 
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
-def requiredLibs = fileTree(dir:'lib', include:['jsr305.jar',
+// TODO : convert these to external dependencies
+def requiredLibs = fileTree(dir:'lib', include:[
```

Comment:
@jsotuyod Could the require libraries use the basic compile directive instead of this additional custom directives?